src/styles/Message.scss

```diff
@@ -370,6 +370,12 @@
       border: 1px solid var(--border);
       margin-left: 0; /* set spacing when unfocused */
 
+      // Fixes emoji display in Chrome https://bugs.chromium.org/p/chromium/issues/detail?id=596223
+      .str-chat__emoji-display-fix {
+        display: inline-block;
+        width: 20px;
```

Comment:
Can this be made `1ch`?

Reply:
I don't why but `1ch` doesn't work, tried it, but it was too narrow

Reply:
Hm, this is a bit dangerous in this form, as people might use larger/smaller fonts in their environment. Can you try 1em? or 1.1em?